recipes/miutil/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "miutil" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f03d1ce4a3fa1f0fc5fffc9e89ff0e757da386b51a6d9b9e6462ed534b97567e
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - cuinfo = miutil.cuinfo:main
+    - mbeautify = miutil.mlab.beautify:main
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - python >=3.4
+    - nibabel
+    - numpy
+    - matplotlib
```

Comment:
Unless a Pyqt-based GUI is needed (this doesn't appear to), better to depend on `matplotlib-base`.